package.json

```diff
@@ -44,7 +44,7 @@
     "contributors:add": "all-contributors add",
     "contributors:generate": "all-contributors generate",
     "dev": "yarn workspace @react95/core storybook",
-    "pretest": "lerna run prebuild",
+    "pretest": "lerna run build",
```

Comment:
This can make tests slower to run, but having `prebuild` here can cause problems if subpackages require `build` to be run to be usable by tests in `core`

Reply:
Ah - yeah, this can be prebuild with the entrypoint changed